pom.xml

```diff
@@ -341,12 +341,12 @@
         <sparkbundle.version>3.5</sparkbundle.version>
         <sparkshim.module.name>spark35</sparkshim.module.name>
         <sparkshim.artifactId>spark-sql-columnar-shims-spark35</sparkshim.artifactId>
-        <spark.version>3.5.2</spark.version>
+        <spark.version>3.5.3</spark.version>
         <iceberg.version>1.5.0</iceberg.version>
         <delta.package.name>delta-spark</delta.package.name>
-        <delta.version>3.2.0</delta.version>
-	<delta.binary.version>32</delta.binary.version>
-	<hudi.version>0.15.0</hudi.version>
+        <delta.version>3.2.1</delta.version>
+        <delta.binary.version>32</delta.binary.version>
+	    <hudi.version>0.15.0</hudi.version>
```

Comment:
Assuming it's unintentional?

Reply:
maybe it's Github diff issue.

![image](https://github.com/user-attachments/assets/fb65a603-514a-4ff7-8ef9-4b06b7185bb3)